src/wp-includes/class-wp-user.php

```diff
@@ -177,14 +180,17 @@ public function __construct( $id = 0, $name = '', $site_id = 0 ) {
 	 *
 	 * @param object $data    User DB row object.
 	 * @param int    $site_id Optional. The site ID to initialize for.
+	 * @param bool   $short_init Optional. Whether to skip initializing capabilities and roles.
 	 */
-	public function init( $data, $site_id = 0 ) {
+	public function init( $data, $site_id = 0, $short_init = false ) {
```

Comment:
This `$short_init` arg isn't used anywhere?

Reply:
I mean, `init()` isn't being called anywhere with this argument provided, right?